packages/lesswrong/server/databaseIndexes/commentsDbIndexes.ts

```diff
@@ -128,6 +128,12 @@ export function getDbIndexesOnComments() {
     WHERE "suggestForAlignmentUserIds" IS DISTINCT FROM '{}';
   `);
 
+  void indexSet.addCustomPgIndex(`
+    CREATE INDEX CONCURRENTLY IF NOT EXISTS "idx_Comments_postId_postedAt_no_parent"
+    ON "Comments" ("postId", "postedAt")
+    WHERE "parentCommentId" IS NOT NULL;
```

Comment:
nit: I think the name is backwards but the condition is correct (or it could be that the condition is backwards
```suggestion
    CREATE INDEX CONCURRENTLY IF NOT EXISTS "idx_Comments_postId_postedAt_with_parent"
    ON "Comments" ("postId", "postedAt")
    WHERE "parentCommentId" IS NOT NULL;
```